buffer/src/main/java/io/netty/buffer/PooledDirectByteBuf.java

```diff
@@ -66,9 +66,24 @@ protected short _getShort(int index) {
     }
 
     @Override
+    protected short _getShortLE(int index) {
+        return ByteBufUtil.swapShort(memory.getShort(idx(index)));
```

Comment:
can we just use `memory.getShortLE(idx(index));`?

Same question for other methods which use `ByteBufUtil` in this class.